Hunting Queries/ASimProcess/imProcess_ExchangePowerShellSnapin.yaml

```diff
@@ -5,17 +5,20 @@ description: |
   Whilst this is a legitimate administrative tool it is abused by attackers to performs actions on a compromised
   Exchange server. Hunt for unusual activity related to this Snapin including it being added on new hosts or by
   new accounts.'
-requiredDataConnectors: []
+requiredDataConnectors:
```

Comment:
When only ASIM sources are involved, at this time, the data connectors list should be empty. 

To explain: 
- "required" is actually always only a recommendation. No actual enforcement. It is mostly for the UX.
- In this case, imProcessCreate supports MDE, Sysmon and potentially others (WindowsEvents coming soon), so SecurityEvent is not the right value to show. We (ab)used the detection name to specify the require source, until a better solution is available.

Reply:
reverted connector changes to original